src/table/internal.tsx

```diff
@@ -532,9 +534,7 @@ const InternalTable = React.forwardRef(
                                   submitEdit={cellEditing.submitEdit}
                                   columnId={column.id ?? colIndex}
                                   colIndex={colIndex + colIndexOffset}
-                                  // Expandable props only apply to the first data column of the table.
-                                  // When present, the cell content is decorated with expand toggles and extra paddings.
-                                  expandableProps={isExpandable && colIndex === 0 ? expandableProps : undefined}
+                                  {...expandableCellProps}
```

Comment:
This change it necessary to not provide empty onExpandableItemToggle when only offset is needed, when a cell does not have the expand toggle but requires an offset.